docs/extras/ecosystem/integrations/arize_llm_observability.ipynb

```diff
@@ -0,0 +1,1765 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Awn1jcNa0lLZ"
+      },
+      "source": [
+        "# Arize <> LangChain Integration\n",
```

Comment:
for consistency with other integrations, lets just call the title `Arize`

Reply:
Done!